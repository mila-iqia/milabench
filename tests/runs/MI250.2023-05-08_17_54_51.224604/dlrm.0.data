{"event": "config", "data": {"dirs": {"base": "/milabench/envs", "venv": "/milabench/envs/venv/torch", "data": "/milabench/envs/data", "runs": "/milabench/envs/runs", "extra": "/milabench/envs/extra/dlrm", "cache": "/milabench/envs/cache"}, "arch": "rocm", "group": "dlrm", "install_group": "torch", "install_variant": "rocm", "run_name": "redizilo.2023-05-08_17:54:51.224604", "enabled": true, "capabilities": {"nodes": 1}, "max_duration": 600, "voir": {"options": {"stop": 60, "interval": "1s"}}, "config_base": "/milabench/milabench/config", "config_file": "/milabench/milabench/config/standard.yaml", "tags": ["nlp", "rl"], "definition": "/milabench/milabench/benchmarks/dlrm", "plan": {"method": "njobs", "n": 1}, "argv": {"--num-batches": 1000, "--data-generation": "random", "--arch-mlp-bot": "512-512-64", "--arch-mlp-top": "1024-1024-1024-1", "--arch-sparse-feature-size": 64, "--arch-embedding-size": "1000000-1000000-1000000-1000000-1000000-1000000-1000000-1000000", "--num-indices-per-lookup": 100, "--arch-interaction-op": "dot", "--numpy-rand-seed": "727", "--print-freq": 999999, "--enable-profiling": true, "--mini-batch-size": 16384, "--test-mini-batch-size": 16384, "--test-num-workers": 0, "--use-gpu": true}, "weight": 1.0, "name": "dlrm", "tag": ["dlrm", "0"], "job-number": 0, "devices": [0, 1, 2, 3, 4, 5, 6, 7]}, "pipe": null}
{"event": "start", "data": {"command": ["voir", "--config", "/milabench/envs/extra/dlrm/voirconf-dlrm.0-0efae956f1553a76c1e03985181900f5.json", "/milabench/milabench/benchmarks/dlrm/dlrm/dlrm_s_pytorch.py", "--num-batches", "1000", "--data-generation", "random", "--arch-mlp-bot", "512-512-64", "--arch-mlp-top", "1024-1024-1024-1", "--arch-sparse-feature-size", "64", "--arch-embedding-size", "1000000-1000000-1000000-1000000-1000000-1000000-1000000-1000000", "--num-indices-per-lookup", "100", "--arch-interaction-op", "dot", "--numpy-rand-seed", "727", "--print-freq", "999999", "--enable-profiling", "--mini-batch-size", "16384", "--test-mini-batch-size", "16384", "--test-num-workers", "0", "--use-gpu"], "time": 1683572236.633738}, "pipe": null}
{"event": "phase", "data": {"name": "init"}, "pipe": "data"}
{"event": "phase", "data": {"name": "parse_args"}, "pipe": "data"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "line", "data": "/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory\n", "pipe": "stderr"}
{"event": "phase", "data": {"name": "load_script"}, "pipe": "data"}
{"event": "line", "data": "Unable to import mlperf_logging,  No module named 'mlperf_logging'\n", "pipe": "stdout"}
{"event": "line", "data": "/milabench/envs/venv/torch/lib/python3.9/site-packages/torch/distributed/distributed_c10d.py:283: UserWarning: torch.distributed.reduce_op is deprecated, please use torch.distributed.ReduceOp instead\n", "pipe": "stderr"}
{"event": "line", "data": "  warnings.warn(\n", "pipe": "stderr"}
{"event": "phase", "data": {"name": "run_script"}, "pipe": "data"}
{"event": "line", "data": "world size: 1, current rank: 0, local rank: 0\n", "pipe": "stdout"}
{"event": "line", "data": "Using 8 GPU(s)...\n", "pipe": "stdout"}
{"event": "line", "data": "time/loss/accuracy (if enabled):\n", "pipe": "stdout"}
{"event": "line", "data": "STAGE:2023-05-08 18:57:28 218400:218400 ActivityProfilerController.cpp:311] Completed Stage: Warm Up\n", "pipe": "stderr"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [33, 65520], "load": 0.0, "temperature": 35.0}, "1": {"memory": [10, 65520], "load": 0.0, "temperature": 40.0}, "2": {"memory": [10, 65520], "load": 0.0, "temperature": 50.0}, "3": {"memory": [10, 65520], "load": 0.0, "temperature": 42.0}, "4": {"memory": [10, 65520], "load": 0.0, "temperature": 43.0}, "5": {"memory": [10, 65520], "load": 0.0, "temperature": 36.0}, "6": {"memory": [10, 65520], "load": 0.0, "temperature": 46.0}, "7": {"memory": [10, 65520], "load": 0.0, "temperature": 44.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [33, 65520], "load": 0.0, "temperature": 35.0}, "1": {"memory": [10, 65520], "load": 0.0, "temperature": 40.0}, "2": {"memory": [10, 65520], "load": 0.0, "temperature": 49.0}, "3": {"memory": [10, 65520], "load": 0.0, "temperature": 42.0}, "4": {"memory": [10, 65520], "load": 0.0, "temperature": 42.0}, "5": {"memory": [10, 65520], "load": 0.0, "temperature": 35.0}, "6": {"memory": [10, 65520], "load": 0.0, "temperature": 45.0}, "7": {"memory": [10, 65520], "load": 0.0, "temperature": 43.0}}}, "pipe": "data"}
{"event": "data", "data": {"task": "main", "gpudata": {"0": {"memory": [33, 65520], "load": 0.0, "temperature": 34.0}, "1": {"memory": [10, 65520], "load": 0.0, "temperature": 40.0}, "2": {"memory": [10, 65520], "load": 0.0, "temperature": 48.0}, "3": {"memory": [10, 65520], "load": 0.0, "temperature": 41.0}, "4": {"memory": [10, 65520], "load": 0.0, "temperature": 42.0}, "5": {"memory": [10, 65520], "load": 0.0, "temperature": 34.0}, "6": {"memory": [10, 65520], "load": 0.0, "temperature": 44.0}, "7": {"memory": [10, 65520], "load": 0.0, "temperature": 42.0}}}, "pipe": "data"}
{"event": "end", "data": {"command": ["voir", "--config", "/milabench/envs/extra/dlrm/voirconf-dlrm.0-0efae956f1553a76c1e03985181900f5.json", "/milabench/milabench/benchmarks/dlrm/dlrm/dlrm_s_pytorch.py", "--num-batches", "1000", "--data-generation", "random", "--arch-mlp-bot", "512-512-64", "--arch-mlp-top", "1024-1024-1024-1", "--arch-sparse-feature-size", "64", "--arch-embedding-size", "1000000-1000000-1000000-1000000-1000000-1000000-1000000-1000000", "--num-indices-per-lookup", "100", "--arch-interaction-op", "dot", "--numpy-rand-seed", "727", "--print-freq", "999999", "--enable-profiling", "--mini-batch-size", "16384", "--test-mini-batch-size", "16384", "--test-num-workers", "0", "--use-gpu"], "time": 1683572258.1024427, "return_code": -11}, "pipe": null}
