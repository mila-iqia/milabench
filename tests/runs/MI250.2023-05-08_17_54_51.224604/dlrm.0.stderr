/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/opt/amdgpu/share/libdrm/amdgpu.ids: No such file or directory
/milabench/envs/venv/torch/lib/python3.9/site-packages/torch/distributed/distributed_c10d.py:283: UserWarning: torch.distributed.reduce_op is deprecated, please use torch.distributed.ReduceOp instead
  warnings.warn(
STAGE:2023-05-08 18:57:28 218400:218400 ActivityProfilerController.cpp:311] Completed Stage: Warm Up
